packages/services/api/src/modules/schema/module.graphql.mappers.ts

```diff
@@ -24,7 +23,22 @@ export type SchemaChangeApprovalMapper = SchemaCheckApprovalMetadata;
 export type SchemaErrorConnectionMapper = readonly SchemaError[];
 export type SchemaWarningConnectionMapper = readonly SchemaCheckWarning[];
 export type SchemaConnectionMapper = readonly SchemaMapper[];
-export type SchemaVersionConnectionMapper = PaginatedSchemaVersionConnection;
+export type SchemaVersionConnectionMapper = Readonly<{
+  edges: ReadonlyArray<{
+    cursor: string;
+    node: SchemaVersion & {
+      organization: string;
+      project: string;
+      target: string;
+    };
+  }>;
+  pageInfo: Readonly<{
+    hasNextPage: boolean;
+    hasPreviousPage: boolean;
+    startCursor: string;
+    endCursor: string;
+  }>;
+}>;
```

Comment:
This change in mapper is required because `PaginatedSchemaVersionConnection` comes from storage and the result is transformed in the [manager](https://github.com/kamilkisiela/graphql-hive/blob/53a0804434979f447c59b0a1c54ed668fc1ab808/packages/services/api/src/modules/schema/providers/schema-manager.ts#L370-L372) to have `organization`, `project` and `target`.

Because of this slightly wrong mapper type, server preset static analysis will make us implement `SchemaVersionConnection`, even when we know we don't have to at runtime.